dashboard/config/levels/custom/spritelab/csc_animal_adaptations_behaviors_wolfpack.level

```diff
@@ -123,7 +123,7 @@
             <arg name="this sprite" type="Sprite"/>
             <description>move a sprite across the screen, reversing direction if it touches the edges</description>
           </mutation>
-          <field name="NAME" id="patrolling">traveling</field>
+          <field name="NAME" id="patrolling">patrolling</field>
```

Comment:
It appears that the author of this level initially considered using the patrolling behavior as a stand-in for "traveling", but later opted for the "swimming left and right" behavior. Since the call block in the toolbox uses the latter, I updated the start blocks to reflect that change and reverted the rename of the former.